src/ui/event.rs

```diff
@@ -114,6 +117,20 @@ pub fn connect_gestures(sender: Sender<Event>, zoom_gesture: &gtk::GestureZoom)
     connect_zoom_gesture(sender, zoom_gesture);
 }
 
+pub fn connect_keybinds(widgets: Widgets, sender: Sender<Event>) {
+    widgets.window().connect_key_press_event(move |_, event| {
+        let keycode = event.keycode().unwrap();
+        if event.state().contains(gdk::ModifierType::CONTROL_MASK) {
```

Comment:
There are three problems here. Unwrap will close the app if there is an error.
event.state().contains() is a bad idea here because for example ctrl+shift+s will also work.
Using keycode numbers is not human readable.
Instead this function should look like this:
`pub fn connect_keybinds(widgets: Widgets, sender: Sender<Event>) {
    widgets.window().connect_key_press_event(move |_, event| {
        if let Some(key) = event.keyval().to_unicode() {
            match key {
                's' if event.state() == gdk::ModifierType::CONTROL_MASK => post_event(&sender, Event::SaveCurrentImage(None)),
                'c' if event.state() == gdk::ModifierType::CONTROL_MASK => post_event(&sender, Event::CopyCurrentImage),
                _ => {}
            }
        }
        gtk::Inhibit(false)
    });
}`

Reply:
I will also remove the SaveCurrentImage shortcut, I didn't notice that it's already handled by an accelerator.

Reply:
Please leave the shortcut and remove the accelerator. Save image accelerator only works when menu is open so a global shortcut is better. I plan to also convert the other accelerators to global shortcuts.